components/module-components/faq/FAQ.module.scss

```diff
@@ -54,21 +54,35 @@
 
 .faqTitle {
   margin-bottom: 1.25rem;
+  font-size: 1.5rem;
+  text-align: center;
 
   @include u.breakpoint(medium) {
     margin-bottom: 1.5rem;
   }
+
+  @include u.breakpoint(xsmall) {
+    font-size: 2.5rem;
+    text-align: start;
+  }
```

Comment:
can you please double check this values for the font size?
I can see that the font-size on mobile is 28(1.75rem) and on desktop 40px(2.5rem)
look into the typography file and check is you could reuse something from there?
See how font size is defined in other sections in order to make it responsive
@MarijaTrajkoska